src/components/VaccinationEligibilityBlock/ButtonBlock.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { EventAction, EventCategory } from 'components/Analytics';
+import { EmailAlertIcon } from 'components/EmailAlertsFooter/EmailAlertsFooter.style';
+import { StyledLinkButton, Wrapper } from './ButtonBlock.style';
+
+const ButtonBlock: React.FC<{ signupLink?: string | null }> = ({
+  signupLink,
+}) => {
+  const sharedTrackingProps = {
+    trackingCategory: EventCategory.VACCINATION,
+    trackingAction: EventAction.CLICK_LINK,
+  };
+
+  return (
+    <Wrapper>
+      <StyledLinkButton
+        to="#shareContainer"
+        {...sharedTrackingProps}
+        trackingLabel="Vaccination alerts"
+        startIcon={<EmailAlertIcon />}
+        aria-label="Vaccination alerts signup"
+      >
+        Get notified when eligibility changes
+      </StyledLinkButton>
+      {signupLink && (
+        <StyledLinkButton
+          href={signupLink}
+          {...sharedTrackingProps}
+          trackingLabel="Where to get vaccinated"
+          endIcon={<OpenInNewIcon />}
+          aria-label="Vaccination information"
```

Comment:
We don't need an aria label when the link has text in any of the children elements, the screen reader will read:

> button: See where and how to get vaccinated

we do need `aria-label` for buttons without text (icon-only buttons) though!